src/coreclr/jit/optimizer.cpp

```diff
@@ -8264,6 +8295,32 @@ void Compiler::optRemoveRangeCheck(GenTree* tree, Statement* stmt)
         gtDispTree(tree);
     }
 #endif
+
+    return check;
+}
+
+//------------------------------------------------------------------------------
+// optRemoveStandaloneRangeCheck : A thin wrapper over optRemoveRangeCheck that removes standalone checks.
+//
```

Comment:
Can you add the Arguments and Returns sections to the header comment (also for the method below).